feature-libs/product-configurator/rulebased/styles/_configurator-product-title.scss

```diff
@@ -101,6 +130,16 @@
           .cx-code {
             color: var(--cx-color-secondary);
           }
+
+          .cx-detail-title,
+          .cx-code,
+          .cx-description {
+            @include forVersion(3.1) {
```

Comment:
change to 3.3 or add an todo

Reply:
The version will be updated accordingly, see #11238